bookmarket-backend/src/main/java/edu/oakland/sophomoreproject/controllers/ListingsController.java

```diff
@@ -79,7 +79,13 @@ public ResponseEntity<Void> createListing(
 			return ResponseEntity.status(403).build();
 		}
 
-		// ... do logic here
+		String title = payload.getTitle();
+		String author = session.getEmail(); // Just used email for now can change to name or something else if needed.
```

Comment:
The author field we want here is actually the author of the book not the listing, which can also be taken from the `payload`, not `session`. But this points out a possible flaw in our modeling, maybe `author` is too ambiguous and we should think about changing it to something like `bookAuthor`. Obviously you don't need to do this, just mentioning it